docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,25 @@
+# Troubleshooting
+
+## Known Problems + Solutions
+
+
+### Unable to delete nodes after uninstalling Karpenter
+If you uninstall Karpenter after creating some nodes with Karpenter, you will be left with nodes that have finalizers that need to be patched out for the node object to be deleted.
```

Comment:
I would invert these statements. The user isn't "left with" anything, and let's be more crisp than "some nodes".

Karpenter adds a [finalizer](link) to nodes that it provisions to support graceful node termination. If Karpenter is uninstalled, these finalizers will cause the API Server to block deletion until the finalizers are removed. 

You can do this manually by patching the node objects:
```


I'd remove any manual instructions for this.